work-practices/version-control/source-control.md

```diff
@@ -31,19 +32,19 @@ Pull requests are the core of a project: they're how changes actually get made,
 
 ## Branches
 
-In general, our projects center around the `main` branch. Continuous integration hooks like CodeDeploy should point to `main`, such that all commits to `main` are deployed to a staging instance, and tagged commits to `main` correspond to releases and are deployed to a production instance.
+In general, our projects center around the `main` branch. Continuous integration hooks from our hosting platforms (e.g., Heroku, Netlify) should point to `main`, such that all commits to `main` are deployed to a staging instance.
```

Comment:
[We switched away from CodeDeploy in 2020!](https://datamade.us/blog/why-were-switching-to-heroku/)